### PR TITLE
Fixes #12225: Do not generate and copy rudder_expected_reports.csv anymore

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/12225

### CODE REVIEW

File: rudder-core/src/main/scala/com/normation/rudder/services/policies/write/AgentSpecificLogic.scala [hunk 1]
```diff
@@ -161,7 +155,7 @@ object CFEngineAgentSpecificGeneration extends AgentSpecificGeneration {
   }
 
   override def write(cfg: AgentNodeWritableConfiguration): Box[List[AgentSpecificFile]] = {
-    writeExpectedReportsCsv(cfg.paths, cfg.expectedReportsCsv, GENEREATED_CSV_FILENAME)
+    Full(Nil)
```
Comment (fanf) [inline rudder-core/src/main/scala/com/normation/rudder/services/policies/write/AgentSpecificLogic.scala]: Could you had a comment referencing the ticket, something like: 
# We believe that expectedReports.csv is not used anymore on agent but still for expecter report generation.
# This is the first step to check that our hypothesis is ok before removing totally: https://www.rudder-project.org/redmine/issues/12225

Comment (VinceMacBuche) [reply rudder-core/src/main/scala/com/normation/rudder/services/policies/write/AgentSpecificLogic.scala]: SHould i write iit this big ? :)

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**